src/gui/MainWindow.cpp

```diff
@@ -552,13 +555,20 @@ void MainWindow::finalize()
 								m_toolBar );
 	controllers_window->setShortcut( Qt::Key_F11 );
 
+    m_countDownClock = new LcdSpinBox(2, this, tr("Count Down Clock"));
+
+    m_countDownClock->setLabel( tr("SEC") );
```

Comment:
I came up with such conversion method:
![image](https://cloud.githubusercontent.com/assets/6829345/24527869/56cc77ec-1561-11e7-99ff-15e11b7bc8ea.png)

The equation (1) is our original algorithm found in TimeDisplayWidget.cpp:112
The equation (2) is to solve the equation (1) given `Tick` value to express `Msec` as a function of `Tick tempo` and `TicksPerTack`
The equation (4) is to ease manual verification of whether the equation (2) works correctly.
As far as I could tell, given the precise value of `ticks` (preserving at least 1 decimal point) can give the very close output (around +- 3 msec at 999 BPM)